core/shared/src/main/scala/fs2/Chunk.scala

```diff
@@ -21,14 +21,15 @@ trait Chunk[+A] { self =>
     val index = iterator.indexWhere(p)
     if (index < 0) None else Some(index)
   }
-  def isEmpty = size == 0
+  def isEmpty: Boolean = size == 0
+  def nonEmpty: Boolean = size != 0
```

Comment:
I was missing this from `ByteVector` and wanted to add this to ease my own migration.  Trivial, but it brightened my day.